README.md

```diff
@@ -1,34 +1,186 @@
-This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next-app`](https://github.com/vercel/next.js/tree/canary/packages/create-next-app).
+<a name="readme-top"></a>
 
-## Getting Started
+[![Next][Next.js]][Next-url]
+[![TypeScript][TypeScript]][TypeScript-url]
+[![Vercel][Vercel]][Vercel-url]
+[![Apache License][license-shield]][license-url]
 
-First, run the development server:
+# ⚡ ️Tigris example app using Next.js 13 - Todo list
 
-```bash
+A simple todo app built on [Next.js 13][Next-url] and [Tigris](https://docs.tigrisdata.com/) 
+using [TypeScript client](https://docs.tigrisdata.com/typescript/), deployed on [Vercel][Vercel-url].
+
+#### Project demo
+
+<a href="https://tigris-nextjs13-starter-kit.vercel.app/">
+    <img src="public/readme/todo_app_screenshot.jpg" alt="Todo web app">
+</a>
+
+https://tigris-nextjs13-starter-kit.vercel.app/
+
+# ⚙️ Deploying the app
+
+You have two options to run this Next.js 13 app:
+<details>
```

Comment:
I think we should move the Vercel Deploy button up top. The hidden sections are easy to miss when someone is glancing over the readme. 

The local development section can be kept as collapsed. But let's keep the Vercel deploy button and the Tigris Cloud deployment section prominent as that is the route we want users to take.